### PR TITLE
Form field changes in Cluster Private Registry CSS changes

### PR DESCRIPTION
**Pull request by neillsom**
**Form fields in Cluster Private Registry & Agent Environment Vars tabs tweaks**
#3184 
Added css styling to make forms consistent height

### CODE REVIEW

File: components/form/SecretSelector.vue [hunk 1]
```diff
@@ -144,6 +144,18 @@ export default {
     display: block;
   }
 
+  & .labeled-select {
+    min-height: 61px;
```
Comment (richard-cox) [inline components/form/SecretSelector.vue:148]: There's a scss variable for form field height (`$input-height`) that should be used here

File: components/form/SelectOrCreateAuthSecret.vue [hunk 2]
```diff
@@ -408,3 +408,9 @@ export default {
     </div>
   </div>
 </template>
+
+<style lang="scss">
+.select-or-create-auth-secret div.labeled-select {
+  min-height: 61px;
```
Comment (richard-cox) [inline components/form/SelectOrCreateAuthSecret.vue:414]: Should also be  `$input-height`

**Review by richard-cox — CHANGES_REQUESTED** [review]
Two v minor tweaks required, others LGTM

**Comment by vincent99** [comment]
@neillsom Please use `git rebase` to update instead of putting merge commits in PRs.

**Pull request merged**